src/main/java/org/broadinstitute/hellbender/utils/genotyper/GroupedEvidence.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.hellbender.utils.genotyper;
+
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GroupedEvidence<EVIDENCE extends Locatable>  extends ArrayList<EVIDENCE> implements Locatable {
```

Comment:
wouldnt' Fragment be the grouped-evidence you need for Mutec?

Reply:
yes, done